gp-includes/things/translation.php

```diff
@@ -163,7 +168,12 @@ public function for_translation( $project, $translation_set, $page, $filters = a
 		$statuses = explode( '_or_', $status );
 		if ( in_array( 'untranslated', $statuses ) ) {
 			if ( $statuses == array( 'untranslated' ) ) {
-				$where[] = 't.translation_0 IS NULL';
+				$empty_plural_translation = '(' . implode( ' OR ', array_map(
+					function( $x ) { return "$x = ''"; },
+					array( 't.translation_0', 't.translation_1', 't.translation_2', 't.translation_3', 't.translation_4', 't.translation_5', )
+				) ) . ')';
+
+				$where[] = "t.translation_0 IS NULL OR (o.plural IS NOT NULL AND $empty_plural_translation)";
```

Comment:
Thinking that we should provide a CLI command to update wrong translations. Since this doesn't affect the number of waiting strings nobody will click the link to see the half-translated strings.